examples/view_generator/variables.tf

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "expiration" {
+  description = "TTL of tables using the dataset in MS"
+  default     = null
+}
+
+variable "project_id" {
+  description = "Project where the dataset and table are created"
+}
+
+variable "time_partitioning" {
+  description = "Configures time-based partitioning for this table"
```

Comment:
Can we choose a sensible default here?

Reply:
The only options are `null` or `DAY`. I think null would be the best default.